.github/workflows/build-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Auth Build Tests
```

Comment:
We can call this `Build Tests` and keep adding to it for any build-related tests.

Reply:
Yes, missed that one!